core/app/views/spree/shared/_purchased_items_table.html.erb

```diff
@@ -48,9 +48,6 @@
         <p class="f-fallback purchase_total purchase_total--label">
           <%= Spree.t(:tax) %>:
         </p>
-        <p class="f-fallback purchase_total--name purchase_total--label">
```

Comment:
@pmilewski why are we removing this?

Reply:
@damianlegawiec I unified displaying VAT amount in [my previous PR](https://github.com/spree/spree/pull/11118/files#diff-2229ed92143ba86489c14ce441ca84466a3794b78f5f67af66167c87029bb903L45) to display it in same way as in [order_summary view](https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/shared/_order_summary.html.erb#L61) but I forgot do delete this _label_.

Reply:
As far as I see, this block:
```
<% if @order.all_adjustments.eligible.tax.exists? %>
  <% @order.all_adjustments.eligible.tax.group_by(&:label).each do |label, adjustments| %>
<%= Spree.t(:tax) %>: <%= label %> <%= Spree::Money.new(adjustments.sum(&:amount), currency: @order.currency) %>
  <% end %>
<% end %>
```
was changed to this:
```
<% if order.additional_tax_total != 0 %>
  <%= Spree.t(:tax) %>: <%= label %> <%= order.display_additional_tax_total.to_html %>
<% end %>
```
here: https://github.com/spree/spree/pull/11118, to fix displaying VAT.
Due to that `<%= label %>` should be removed indeed (it's no longer needed because `label` was a block variable.)